src/nanoemoji/nanoemoji.py

```diff
@@ -151,6 +155,10 @@ def _ufo(family: str, upem: int) -> ufoLib2.Font:
     space.width = upem
     ufo.glyphOrder = [".notdef", ".space"]
 
+    if not keep_glyph_names:
+        # use 'post' format 3.0 for TTFs, shaving a kew KBs of unneeded glyph names
+        ufo.lib[ufo2ft.constants.KEEP_GLYPH_NAMES] = keep_glyph_names
```

Comment:
why do we need an if? - just always set = keep_glyph_names

Reply:
yeah, that works too. I'll remove the if